### ISSUE TITLE
500 ISE when doing a GET for a joke ID that doesn't exist

### ISSUE THREAD

**Issue by SpikesDivZero** [chucknorris-io/chuck-api#16]
Hi and thanks for creating this! I'm using this API as an endpoint to help someone learn, and I stumbled across an unexpected variance when interacting with the API:

Doing a GET for an ID that doesn't exist (but otherwise passes validation) results in a 500 Internal Server Error with no response body.

The request being sent to the server (only including explicitly defined headers in our client):
```
GET /jokes/no_such_id_aaaaaaaaaaa HTTP/1.1
Accept: application/json
```

The response headers only had the standard headers, along with CloudFlare and Access-Control-Allow-*.

Thanks for the time in creating this! :)

**Comment by matchilling** [comment]
Hi @SpikesDivZero, thanks for taking the time and reporting this issue. I’ll take a look and create a fix for the bug once I’m back from holidays 🏝

**Comment by matchilling** [comment]
Fixed in [684ed8616fc52b7d0129305c60168e4a0e77aa07](https://github.com/chucknorris-io/chuck-api/commit/684ed8616fc52b7d0129305c60168e4a0e77aa07).
